spec/services/av_characterizer_service_spec.rb

```diff
@@ -4,9 +4,27 @@
 
 RSpec.describe AvCharacterizerService do
   let(:service) { described_class.new }
+  let(:status) { instance_double(Process::Status, success?: true) }
+  let(:volume_detect) do
+    <<~VOLUME
+      Lots of stuff here
+      [Parsed_volumedetect_0 @ 0x6000012f00b0] n_samples: 28525216
+      [Parsed_volumedetect_0 @ 0x6000012f00b0] mean_volume: -24.2 dB
+      [Parsed_volumedetect_0 @ 0x6000012f00b0] max_volume: -4.7 dB
+      More unused stuff here
+    VOLUME
+  end
+  let(:track_info) { 'track info' }
+  let(:ffmpeg_command) { "ffmpeg -i #{filepath} -af 'volumedetect' -vn -sn -dn -f null /dev/null" }
+  let(:ffprobe_command) { "ffprobe -i #{filepath} -show_streams -select_streams a -loglevel error" }
 
   before do
-    allow(Open3).to receive(:capture2e).and_return([output, status])
+    allow(Open3).to receive(:capture2e).with('mediainfo', '-f', '--Output=JSON', /.*/).and_return([output, status])
+    allow(Open3).to receive(:capture2e).with('mediainfo --Version').and_return([output, status])
+    allow(Open3).to receive(:capture2e).with(%r{ffmpeg -i .* -af 'volumedetect' -vn -sn -dn -f null /dev/null})
+                                       .and_return([volume_detect, status])
+    allow(Open3).to receive(:capture2e).with(/ffprobe -i .* -show_streams -select_streams a -loglevel error/)
+                                       .and_return([track_info, status])
```

Comment:
Is there a reason why we don't want to actually invoke ffmpeg here?

Reply:
seems to not really be needed to verify our code does what it should, and if we do, it requires anything running the tests to have it installed (like circleci) ... may also slow down the tests a bit to have it actually execute each tine?

Reply:
Unfortunately it also means if ffmpeg changes in some way the unit tests won't tell us. I guess I tend to err on the side of mocking too little...

Reply:
yes, that is also true -- it's hard to find the right medium

we can do some follow on work if you think it's useful to improve the tests